cdap-data-fabric/src/test/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueTest.java

```diff
@@ -317,8 +305,22 @@ protected void verifyConsumerConfigIsDeleted(QueueName... queueNames) throws Exc
     }
   }
 
+  private ConsumerConfigCache getConsumerConfigCache(QueueName queueName) throws Exception {
+    TableId tableId = TableId.from(((HBaseQueueAdmin) queueAdmin).getConfigTableName(queueName));
```

Comment:
Make `HBaseQueueAdmin.getConfigTableName` to `HBaseQueueAdmin.getConfigTableId`.